FindMe/Model/Contact.m

```diff
@@ -7,17 +7,39 @@
 //
 
 #import "Contact.h"
+@import Contacts;
+@import ContactsUI;
 
 @implementation Contact
 
-    @dynamic telephoneNumber;
-    @dynamic name;
-    @dynamic user;
-    @dynamic email;
+@dynamic telephoneNumber;
+@dynamic name;
+@dynamic user;
+@dynamic email;
+@dynamic lastName;
 
 + (nonnull NSString *)parseClassName
 {
     return @"Contact";
 }
 
++ (void)uploadContacts: (NSArray<CNContact*> *)contacts withCompletion: (PFBooleanResultBlock  _Nullable)completion
+{
+    if(contacts) {
+        for (CNContact *contact in contacts) {
+            Contact *newContact = [Contact new];
+            newContact.name = contact.givenName;
+            newContact.user = [PFUser currentUser];
+            if(contact.givenName) {
```

Comment:
spacing. Also I don't understand this check. You've already used `contact.givenName` without checking for it's existence. Why here and not there?

Reply:
Also you're not checking for familyName.

Reply:
Made a mistake, it was a check for family name.